akka-http-core/src/main/scala/akka/http/impl/engine/ws/UpgradeToWebSocketLowLevel.scala

```diff
@@ -5,6 +5,7 @@
 package akka.http.impl.engine.ws
 
 import akka.annotation.InternalApi
+import akka.http.impl.engine.server.InternalCustomHeader
```

Comment:
Extra import?

Reply:
This one is needed because `akka.http.impl.engine.ws.InternalCustomHeader` was promoted to `akka.http.impl.engine.server.InternalCustomHeader`

Reply:
Ah, I see. Thanks.